userguide/plugins.rst

```diff
@@ -378,14 +385,12 @@ GitHub repository.
    `GitHub Guides <https://guides.github.com/>`__.
 
 
-On the local copy of :file:`iocage-ix-plugins`, create a new file for
-the plugin to be added to %brand%. The naming convention is
-:file:`pluginname.json`. For example, the :guilabel:`Transmission`
-plugin has a .json file named :file:`transmission.json`.
-
-Add fields to this .json file.
-:numref:`table %s <plugins-plugin-jsonfile-contents>` lists and
-describes each required entry.
+On the local copy of :file:`iocage-ix-plugins`, create a new JSON file
+for the %brand% plugin with the fields described in
```

Comment:
This does not explain what the file does. The naming convention should be mentioned next, because the user has to enter a name when they create the file.
`` A single JSON file describes the plugin, the packages it requires for operation, and other installation details. This file is named :samp:`{pluginname}.json`. For example, the Transmission(link) plugin file is named :file:`transmission.json`. ``